Sources/AWSSDKSwiftCore/AWSShapes/TimeStamp.swift

```diff
@@ -43,7 +43,9 @@ public struct TimeStamp {
     }
     
     /// date formatter for outputting dates
-    private static var defaultDateFormatter : DateFormatter {
+    private static var defaultDateFormatter : DateFormatter = createDefaultDateFormatter()
```

Comment:
I would change the `var` to `let` in this case. Otherwise this is unsafe. Especially considering threadSafety.